tests/unit/CorrectCompanyName.spec.ts

```diff
@@ -1,21 +1,18 @@
-import Vue from 'vue'
 import Vuetify from 'vuetify'
 import flushPromises from 'flush-promises'
 import { mount, Wrapper } from '@vue/test-utils'
 import CorrectCompanyName from '@/components/correct-name/CorrectCompanyName.vue'
 
-Vue.use(Vuetify)
-
 function getLastEvent (wrapper: Wrapper<CorrectCompanyName>, name: string): any {
-  const eventsList: Array<any> = wrapper.emitted(name)
+  const eventsList = wrapper.emitted(name) as Array<any>
   if (eventsList) {
-    const events: Array<any> = eventsList[eventsList.length - 1]
+    const events = eventsList[eventsList.length - 1] as Array<any>
     return events[0]
   }
   return null
 }
 
-xdescribe('CorrectCompanyName', () => {
```

Comment:
`xdescribe` doesn't work with Vite?

Reply:
Not out of the box.

`xdescribe` is an alias to `describe.skip` when using Jest directly. Maybe it's possible to add it to Vitest but I didn't look into it.

Reply:
OK, noted. Resolving.